### PR TITLE
Add AWX Operator stack

### PR DESCRIPTION
**Pull request by mamercad**
## BACKGROUND
Add the [AWX Operator](https://github.com/ansible/awx-operator) stack.

-----------------------------------------------------------------------

## Changes
This is the first release, there are no changes.

-----------------------------------------------------------------------

## Checklist
- [x] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
------------------------------------------------------------------------

Reviewer: @marketplace-eng

```bash
❯ ./stacks/awx-operator/deploy.sh
"awx-operator" has been added to your repositories
Release "awx-operator" does not exist. Installing it now.
NAME: awx-operator
LAST DEPLOYED: Sun Dec  4 21:38:13 2022
NAMESPACE: awx
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
AWX Operator installed with Helm Chart version 1.1.1
```

```bash
❯ ./stacks/awx-operator/upgrade.sh
"awx-operator" already exists with the same configuration, skipping
Release "awx-operator" has been upgraded. Happy Helming!
NAME: awx-operator
LAST DEPLOYED: Sun Dec  4 21:39:09 2022
NAMESPACE: awx
STATUS: deployed
REVISION: 2
TEST SUITE: None
NOTES:
AWX Operator installed with Helm Chart version 1.1.1
```

```bash
❯ kubectl -n awx get all
NAME                                                   READY   STATUS    RESTARTS   AGE
pod/awx-operator-controller-manager-85575b5c5c-tfd4v   2/2     Running   0          69s

NAME                                                      TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
service/awx-operator-controller-manager-metrics-service   ClusterIP   10.43.175.107   <none>        8443/TCP   69s

NAME                                              READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/awx-operator-controller-manager   1/1     1            1           69s

NAME                                                         DESIRED   CURRENT   READY   AGE
replicaset.apps/awx-operator-controller-manager-85575b5c5c   1         1         1       69s
```

```bash
❯ ./stacks/awx-operator/uninstall.sh 
release "awx-operator" uninstalled
```

```bash
❯ kubectl -n awx get all            
No resources found in awx namespace.
```

### CODE REVIEW

File: stacks/awx-operator/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+################################################################################
+helm repo add awx-operator https://ansible.github.io/awx-operator/
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="awx-operator"
+CHART="awx-operator"
+CHART_VERSION="1.1.1"
```
Comment (scott) [inline stacks/awx-operator/deploy.sh:16]: Looks like this needs to be updated to 1.1.2 for this to work.  The current version is not found.

**Review by scott — CHANGES_REQUESTED** [review]
Hello from DigitalOcean, sorry for the delay.  Can you please update your chart version and resubmit?

File: stacks/awx-operator/deploy.sh (hunk 1 above)
Comment (mamercad) [reply stacks/awx-operator/deploy.sh:16]: Surely, might be related to [this](https://github.com/ansible/awx-operator/issues/1136).

**Comment by mamercad** [comment]
> Hello from DigitalOcean, sorry for the delay. Can you please update your chart version and resubmit?

This is done, could you please re-review?

**Review by scott — APPROVED** [review]
LGTM 👍  although I will bump the version yet again to 1.1.3 :)

**Pull request merged**